### PR TITLE
add(search users page close #39)

### LINKED ISSUE #39: Add search functionality
Create search for users.

It should be field/page somewhere that accepts user name/surname and shows a list of possible matches - users.

### PR DESCRIPTION
**Pull request by slava89**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,6 +27,7 @@
     "express-session": "^1.14.0",
     "gulp": "^3.9.1",
     "mongoose": "^4.5.7",
+    "mongoose-text-search": "0.0.2",
```
Comment (andrewdacenko) [inline package.json]: remove unused dependency

File: server.js [hunk 2]
```diff
@@ -128,6 +120,39 @@ app.get('/auth/google/callback',
...
+//     } else {
+//       console.log("found!");         
+//       res.json(data);
+//     }    
+//   });    
+// });
+
+
+//search using regex
+app.post('/searchUsers', function (req, res) {
```
Comment (andrewdacenko) [inline server.js]: ensure authenticated

**Comment by andrewdacenko** [comment]
@slava89 fix please

File: server.js [hunk 3]
```diff
@@ -120,24 +120,6 @@ app.get('/auth/google/callback',
...
-//     } else {
-//       console.log("found!");         
-//       res.json(data);
-//     }    
-//   });    
-// });
-
-
 //search using regex
 app.post('/searchUsers', function (req, res) {
```
Comment (andrewdacenko) [inline server.js]: ensure authenticated

**Comment by slava89** [comment]
wait, didnt see the comment, will fix

**Pull request merged**